static/js/publisher/utils/filterAvailableStores.ts

```diff
@@ -0,0 +1,9 @@
+import type { AvailableStores } from "../types/shared";
+
+function filterAvailableStores(stores: AvailableStores) {
+  const PUBLIC_STORES = ["LimeNET", "LimeSDR", "orange-pi", "china"];
+
+  return stores.filter((store) => !PUBLIC_STORES.includes(store.id));
```

Comment:
I guess I lack a bit of context here. What is function supposed to do? 
From what I see it filters **out** the public stores from a list of stores it's given. But why? Aren't public stores "available"?

Reply:
Ok. Maybe the "PUBLIC_STORES" mislead me a bit. It seems that those are the brand stores where you can't register your snap in? That's why they are filtered out?

Reply:
Yes, that's correct. We shouldn't be showing these brand stores, even though they are returned by the API.